https://invent.kde.org/plasma/plasma-workspace/-/commit/432d7c4b51c5a1f17af327d770266b3fe81e5ae5

From 432d7c4b51c5a1f17af327d770266b3fe81e5ae5 Mon Sep 17 00:00:00 2001
From: Nicolas Fella <nicolas.fella@gmx.de>
Date: Tue, 27 Sep 2022 18:04:51 +0200
Subject: [PATCH] [applets/systemtray] Don't manually destroy innerContainment

It's already destroyed by the corona

BUG: 420245
(cherry picked from commit 7baa4c221e45f161adf4e00d4cf0e36d6436e90c)
--- a/applets/systemtray/container/systemtraycontainer.cpp
+++ b/applets/systemtray/container/systemtraycontainer.cpp
@@ -20,9 +20,6 @@ SystemTrayContainer::SystemTrayContainer(QObject *parent, const KPluginMetaData
 
 SystemTrayContainer::~SystemTrayContainer()
 {
-    if (destroyed()) {
-        m_innerContainment->destroy();
-    }
 }
 
 void SystemTrayContainer::init()
GitLab
